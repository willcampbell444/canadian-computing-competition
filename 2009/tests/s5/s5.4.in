30000
1000
1000
67 1866 9044 529
6 21289 7146 835
90 12305 2841 603
68 22508 4193 666
22 16197 2134 752
45 14259 9402 964
29 1716 3216 819
21 2078 4551 593
75 5788 5524 889
20 15933 2813 666
24 1109 4682 759
13 5543 9521 709
24 86 2303 909
59 8168 6220 689
60 16788 1765 568
67 22780 9480 879
29 11194 1917 512
57 17148 7399 847
75 17733 1598 714
37 19939 7283 577
6 7560 6826 718
86 22765 6358 867
39 29945 817 961
35 29958 7024 972
41 28363 8115 995
93 1650 4972 628
99 3319 5490 971
77 16735 1713 860
51 26835 3313 632
84 5636 5247 872
77 2022 9220 706
3 5012 7368 509
60 3696 3264 833
58 9505 6147 588
89 5456 2239 930
67 2218 7267 784
10 15117 3652 687
7 16593 4614 502
45 25235 2140 537
30 5047 2691 797
11 15941 6270 824
67 12321 6439 938
55 7008 4725 972
56 873 621 670
29 13766 4592 776
82 29187 2730 946
73 18526 7006 676
94 4102 528 519
7 11604 5061 518
10 16026 2076 839
73 28916 9280 904
36 15280 5483 609
74 9216 1737 629
34 16407 1949 607
44 25106 1903 674
29 10237 1713 560
39 20735 8464 749
71 6600 9264 968
56 16295 971 996
35 27043 2893 654
81 28060 6432 823
85 21915 8915 511
61 23080 9703 602
79 28589 8923 579
76 5617 9556 509
35 22847 4627 639
87 1442 7873 582
94 22221 4893 919
53 14874 8414 742
52 2289 9402 599
42 15565 2737 512
44 23620 4662 684
19 6520 5964 680
88 5305 866 655
81 6674 2674 710
24 4074 844 730
29 12076 4787 879
2 8880 3189 939
19 26743 4754 583
80 27662 3312 949
55 15150 2230 801
12 21042 3539 750
81 4792 7238 552
91 25104 1633 813
5 519 6662 515
81 16606 2968 746
94 17799 6345 694
13 8974 6465 561
46 15167 3913 845
80 8757 6717 929
91 9612 359 627
83 13482 7670 616
96 17287 6535 598
96 1876 9396 602
57 17789 8715 944
81 5069 3764 837
17 21479 6072 589
25 25348 3607 647
54 10768 5951 987
76 6288 2470 526
2 28523 9866 600
14 13349 8565 671
63 26470 9804 775
91 23496 6593 603
24 15059 6313 626
98 22101 4880 560
42 2352 7448 864
19 1069 3926 968
87 21472 1008 565
66 5201 2870 692
15 18780 315 661
89 26157 1033 912
60 4013 1413 600
87 19557 4771 972
3 27428 2864 684
67 22273 448 751
29 23254 9275 938
28 8946 4631 768
39 1894 5818 827
4 19127 6098 704
39 27815 4001 616
90 9626 4481 955
85 17844 51 835
90 9751 7128 901
80 16171 1296 713
88 27760 2731 661
49 25726 3054 545
74 904 9547 907
1 22313 768 695
26 25343 8945 792
94 8710 7222 842
14 2636 2236 802
81 1346 8463 668
70 21668 1291 773
15 418 6296 609
94 24241 1211 948
81 10995 6021 890
83 2307 2570 990
76 13777 6262 874
52 19041 7855 532
41 23732 660 569
3 10362 7216 842
94 29169 7788 995
92 18788 5348 951
2 6373 5846 961
95 14985 8817 822
82 10222 3459 750
44 26333 1554 876
94 17636 5726 1000
60 11848 9944 666
31 10453 9966 628
89 11122 8862 777
98 8243 8859 725
78 21593 860 728
66 21259 6913 691
32 18361 1644 780
23 20677 8110 858
25 18534 5019 845
19 11755 4723 891
54 4108 3166 930
53 5904 5389 926
70 14982 7059 928
93 1887 9570 661
91 11924 8091 534
91 11765 4439 946
65 27467 7812 623
65 8384 2259 636
77 15570 7113 652
71 11582 3377 544
33 18792 390 820
40 3125 3067 655
73 25283 6437 628
73 28215 1885 999
32 6837 5961 697
57 12248 4906 605
57 5685 2472 914
61 13779 9033 642
87 4645 136 681
78 27726 6167 951
18 2151 5707 668
3 4990 5484 907
54 9434 3978 750
16 5110 7583 602
90 20824 4114 849
44 18803 5163 912
27 6630 3669 524
35 25404 1140 568
21 12605 6737 863
72 223 6201 623
78 15725 959 976
89 1093 5271 688
96 7730 2937 833
46 26040 7478 870
98 18572 8920 831
91 16984 9079 876
7 27211 7971 733
61 578 1705 886
81 22938 1505 941
76 13721 928 692
71 11972 4012 650
64 6968 6737 834
7 913 3738 531
50 17497 7831 718
73 4008 3677 688
85 24785 3765 992
34 5765 4209 590
57 23009 6607 956
71 13680 9843 786
25 14986 3884 842
7 19483 9154 890
85 13908 1361 998
33 23109 8977 889
92 7517 1397 851
75 21277 3113 967
54 18583 7973 969
11 20725 8266 617
54 10387 4743 613
57 1960 4289 960
87 7493 2519 699
35 16819 2899 985
64 478 3889 845
87 25938 9856 819
60 14893 7943 892
50 5670 3820 838
41 22303 4216 530
74 12797 2657 776
16 29413 7796 832
86 1002 264 741
88 5249 704 770
28 25727 3090 937
98 2688 8172 761
20 28399 2695 939
92 27661 1281 617
1 11218 8221 518
42 12516 1964 950
33 15479 4242 564
88 22773 4146 552
29 28316 8651 616
11 26326 7632 821
31 16430 818 915
47 11275 2088 629
4 5347 3013 603
88 20445 8488 825
86 4573 1211 935
70 4474 4361 502
67 29698 6785 547
61 3829 5001 847
84 3144 4483 803
6 22782 6502 552
37 10409 7738 613
81 7176 3794 694
82 26318 7440 628
42 6451 600 944
81 7060 1368 783
29 6742 7603 726
1 15312 9312 998
18 7906 3772 506
70 8406 6296 924
46 19450 3670 518
76 1127 3028 681
96 15818 7043 762
5 7377 7759 780
14 1385 7988 765
63 18514 7862 706
16 23032 8116 992
53 20385 1491 936
1 7950 2960 869
17 28877 5840 804
35 5813 5534 856
86 2492 9918 922
67 11651 3827 510
19 13129 8679 776
73 13534 2145 638
68 12378 578 948
63 17069 7167 807
14 1379 6117 607
63 26301 5167 842
92 21301 1884 583
46 12510 7249 871
56 5765 2977 819
100 14529 423 501
78 18468 419 772
94 2540 2332 936
77 6208 761 536
44 21139 9058 961
73 9987 8346 503
12 18019 822 594
91 15367 8011 838
7 26490 7879 636
40 29485 1706 504
83 4185 7752 548
7 13959 7273 646
53 3002 7335 751
33 2626 241 740
78 12244 3369 688
17 17068 5671 982
65 20024 6857 709
65 11856 2934 718
81 10848 8522 735
34 5339 6303 932
99 1639 9881 607
71 17531 1949 531
92 300 9862 611
49 23060 6305 941
44 22713 5467 627
29 21961 2973 942
91 12409 5887 608
76 15424 9476 817
72 18474 8519 790
39 24094 1129 677
74 29082 4261 834
90 7923 9780 963
70 9285 3003 576
66 16398 3013 907
73 8256 7491 904
49 26348 1726 608
73 9920 6375 945
62 15464 4011 528
25 9635 8689 610
14 20014 9250 538
54 27835 1251 896
50 10019 9804 895
100 20722 8014 741
52 28448 5484 656
69 28277 5007 741
46 10130 5826 936
74 822 2582 636
72 18639 4085 525
82 22425 4845 836
48 9732 5567 556
75 16847 2412 568
66 22527 5410 726
100 26586 4790 864
29 26697 7762 894
59 9554 9324 683
91 2830 4516 704
71 19973 3969 516
15 13570 5425 548
85 29189 9701 508
77 21551 1605 913
66 12576 5393 721
89 4347 7079 555
96 28647 7802 993
59 24438 7330 944
80 13882 2291 581
73 14594 9188 965
83 10057 1170 571
73 13759 5404 553
53 8591 7722 799
53 15793 4793 809
79 28348 9599 836
50 1090 6037 618
12 21529 1987 547
95 7336 1637 625
11 14678 1512 607
1 29636 8980 703
34 3205 8487 974
49 16287 6668 662
56 15980 1265 639
70 22805 4928 623
15 7681 5871 891
16 15562 1900 658
13 15627 9765 759
51 3665 8204 526
100 27337 7259 839
7 15348 7202 569
35 9462 2020 960
89 17456 6960 747
11 19356 2781 802
13 27279 8734 503
32 24778 3023 655
8 5383 4015 518
83 1707 6106 663
4 21290 1276 545
31 23790 933 585
74 11672 7571 721
34 6537 5768 732
49 7199 8187 795
40 10332 1256 571
83 14647 8319 769
28 21366 101 784
94 10184 1452 610
29 19377 2440 757
1 9555 9815 570
20 9887 8537 926
27 14520 9309 757
32 23266 9792 912
52 3315 3663 917
77 18232 1677 968
42 16623 5164 686
12 21623 2286 591
27 17313 3268 750
75 1598 924 619
50 15553 8396 987
87 7281 8692 819
100 4991 6563 933
59 5828 7728 774
30 28753 4310 988
1 2442 227 652
73 11779 1481 755
42 24099 613 656
15 25391 1071 844
4 16453 5458 672
43 13355 2033 618
3 26554 2728 664
35 14785 4985 984
38 36 6650 884
38 6862 6259 831
37 15979 8619 542
33 13251 9842 962
12 18736 3042 595
14 23955 6274 998
6 7516 5683 623
100 21093 7106 517
59 29537 9557 877
86 24896 9379 793
22 923 9908 709
41 25154 7039 946
78 7104 7335 580
56 12591 7500 534
85 24614 6109 769
86 6036 5876 610
29 16545 2943 685
81 16015 9907 673
63 21754 8976 666
93 24355 4849 813
84 10822 7767 851
89 27540 951 720
82 28625 4602 721
94 15020 8499 710
57 26222 6889 584
24 25989 8641 707
41 17039 1793 876
4 18785 491 853
75 2087 8527 703
41 6615 568 721
91 5975 9664 982
87 22436 3766 751
67 26795 5184 761
25 8043 8235 597
82 3865 7782 729
80 24137 188 825
26 19154 6584 606
86 12618 7421 905
61 14429 170 858
49 15499 4919 937
22 28516 7925 725
8 29998 3649 779
14 27941 4752 990
67 29193 3745 862
7 12144 3839 503
75 11645 7362 965
93 4135 3167 557
65 9542 7069 640
7 22038 8887 629
7 19766 1797 818
9 2472 7746 748
89 12927 9611 986
79 9395 7776 862
38 27353 4497 682
55 10583 6351 815
3 9319 985 823
53 27209 2526 736
89 26211 6611 742
70 7097 826 884
12 3358 7211 698
77 2590 9123 944
9 22567 4593 995
6 8115 5561 984
11 19108 8523 976
33 14115 966 640
78 9573 4903 994
56 19599 6153 858
90 7819 6524 895
57 11167 1592 911
36 3918 7737 553
60 14193 6213 585
8 25122 2893 817
65 8541 1554 819
69 4394 3870 912
80 9676 4385 986
9 24070 269 947
5 22966 5601 753
8 17192 5000 524
82 19519 6372 766
6 28390 7050 535
90 14370 439 727
79 20110 2244 894
52 2111 7464 754
36 24112 7006 631
94 7892 8541 971
77 12453 4681 811
54 26658 5525 819
40 3559 1063 962
74 2029 3395 828
47 11304 6428 542
58 14550 8651 775
67 8845 1445 797
100 7612 3962 738
90 4328 6490 671
699 11195 16884 886
712 3590 12780 684
425 3137 2416 432
734 571 29678 308
502 13343 19789 354
221 15621 7481 532
149 24621 28057 127
385 13490 18726 548
669 28436 24504 685
171 25732 21909 288
102 5947 1550 295
512 3187 12187 507
472 14014 28447 840
398 6692 1584 220
216 5547 27093 517
896 25953 15726 112
211 27916 18180 372
406 19734 18911 412
173 4859 999 714
888 6036 12896 549
698 17359 15412 179
603 18096 15657 934
677 3170 19687 614
804 27573 1366 581
921 10367 24888 209
352 326 16107 387
679 6524 6360 979
860 23567 6692 782
346 1909 24500 660
745 19300 12849 679
82 10421 15730 701
451 26237 16657 743
23 17723 2271 685
185 12374 22054 554
870 25863 25213 637
472 5081 20893 761
88 10817 11658 558
397 2832 28168 197
231 16830 27184 889
133 1033 4107 749
427 8155 19440 973
796 6963 19993 408
290 3665 29476 930
367 12113 15502 258
133 1454 13548 285
37 26882 1155 449
838 20619 11294 890
854 17971 3567 581
618 27280 15885 411
486 20765 28168 957
727 4127 23709 318
609 19093 24834 925
621 16094 6473 921
73 15010 4340 684
518 22910 29572 137
93 24371 1233 183
65 22062 20742 609
364 16903 26778 295
130 23091 21930 206
492 1091 15590 820
264 3113 9628 481
546 5336 18205 610
436 22063 17260 501
537 19528 4842 877
732 602 23230 538
725 4096 18212 267
856 13854 10794 636
153 16699 9712 572
988 11416 18679 479
737 24534 5185 284
529 7172 14937 501
222 1066 1956 285
111 1306 2412 692
109 18835 222 474
361 4316 29333 990
462 8513 19379 384
165 17450 25375 229
788 4901 28061 336
718 19316 6565 282
818 28940 13610 926
976 23280 29387 103
286 12437 20112 397
840 4646 16808 605
999 302 25261 770
230 28163 16160 943
70 24017 8446 837
15 9788 11258 990
519 20832 11592 317
319 16847 21665 423
785 956 24608 928
208 10624 20624 286
957 3206 26244 415
167 24186 24827 679
323 6589 17097 734
654 23337 11427 838
128 4524 14580 954
263 26798 9243 283
94 15399 11318 456
872 21301 18611 205
29 23828 16442 499
710 7548 21380 877
156 23384 5314 119
668 10161 20163 663
927 7795 15898 192
114 3541 3863 558
752 559 5842 396
610 18623 22651 660
292 11991 22751 916
438 3353 18040 368
887 17946 13248 473
751 1554 23289 707
604 10677 27067 694
690 19069 17809 943
466 12793 11034 255
200 5428 25774 185
89 11843 21570 870
979 24526 1493 455
88 10217 24358 982
422 24190 14300 229
815 9377 15105 621
940 24856 15869 634
555 5758 19230 171
984 10106 17604 815
908 28072 22024 202
775 5238 24898 394
700 16215 10326 420
747 23474 10306 375
407 20561 4217 422
365 28347 16352 795
830 25272 20947 415
241 26813 12667 388
874 8500 11682 529
957 8001 15842 286
116 22657 20468 190
943 22862 29929 870
625 2350 14731 231
955 22342 6654 541
790 25784 10526 751
73 23030 2750 393
370 21947 5903 585
158 24678 2218 417
834 6273 19241 313
146 26777 5149 315
151 1770 19342 459
323 24105 16361 873
120 20112 6264 227
725 28544 17077 689
341 8446 25784 173
300 22498 13029 136
271 15134 11001 157
217 5416 21046 507
559 25367 7826 635
779 10881 18074 300
603 21250 11114 563
912 25327 10845 288
70 2252 22982 483
703 4370 26123 147
180 10325 2922 922
798 11748 20000 312
981 12356 1818 330
703 10759 4172 988
470 1305 5967 483
435 29733 14040 301
353 15504 21930 907
9 6407 23051 836
138 21540 14492 527
157 5935 5342 876
391 29489 22624 952
511 10093 25008 629
759 332 28525 759
543 1545 29526 785
538 12896 8962 751
1000 22546 23254 173
616 22489 22789 589
474 20124 17746 423
998 12865 14407 117
522 13860 13285 364
494 633 8262 410
710 7037 14930 359
202 12247 3831 777
730 20553 25159 489
304 16345 11138 729
585 5777 13010 816
996 23582 11304 733
63 28869 23258 754
295 20909 26785 836
606 19356 1985 930
220 25159 23162 680
679 4949 588 898
919 15982 24121 952
898 16503 19239 914
756 4245 5942 103
916 24170 15850 629
643 18562 1625 914
305 20600 12372 274
968 1525 790 671
572 25646 1448 369
583 29491 15924 918
191 27480 13240 237
361 2946 5876 233
802 2628 19625 189
270 796 28791 945
976 10575 22125 706
326 26296 4208 705
745 4249 22062 475
8 2679 775 120
568 5198 22560 436
422 9280 6949 234
857 9575 16209 361
554 27127 16692 548
973 29881 16634 769
481 25650 16866 855
92 28225 3085 621
424 13099 25106 489
999 6343 308 966
454 17773 24046 842
328 22663 108 381
889 28233 21934 121
399 2861 27732 693
813 11879 17171 193
749 24904 17088 661
476 3525 10162 711
494 5826 14457 701
721 4300 21543 141
69 25400 806 149
646 28957 16818 402
442 5044 19350 950
143 19255 20079 831
791 15244 26906 229
568 17266 8410 516
826 21062 10700 886
41 18898 14417 693
775 27109 21395 874
83 7772 24347 934
616 6524 27577 786
169 24111 14238 488
207 1959 7546 892
106 6774 15126 321
412 16284 4473 520
572 3805 29643 899
167 2698 171 182
267 24249 26869 430
723 20234 7374 335
597 4243 27154 706
817 1155 20763 697
796 1275 8108 977
366 21208 15196 867
328 5289 21405 794
957 19494 29050 276
442 8575 8524 907
824 6516 10335 919
849 3773 14910 652
87 5017 12623 258
674 21882 14049 707
698 8391 6514 652
80 15025 9690 804
176 11388 18540 517
182 23617 11289 412
274 17358 26806 271
902 21474 6262 893
263 11988 7939 356
389 29288 5681 885
466 23086 29103 310
725 13545 18732 756
591 24900 4520 727
368 18124 16532 346
943 994 25361 851
666 18196 17505 610
442 14280 22778 157
66 2830 645 305
951 531 19506 344
354 14675 1766 876
308 12944 11718 511
791 12028 24734 341
510 7085 23667 700
796 8527 18400 487
220 5237 24077 116
319 22285 20305 524
59 29269 22447 584
304 26412 16673 194
310 6964 16857 269
26 16452 21786 127
849 21887 14188 663
984 14644 23218 369
293 9203 26946 270
684 22611 25078 301
600 7094 2950 806
395 14255 17985 627
395 22704 16561 217
274 13772 4271 760
212 15123 10838 444
993 1195 23320 171
350 24285 9599 350
593 29386 21240 987
357 4544 6937 931
956 4751 22038 888
279 2350 27870 268
357 10553 21564 514
202 8920 6690 332
493 22546 15777 144
800 6195 12562 314
612 29392 1689 382
450 6968 2053 810
804 18405 25018 985
541 7434 895 676
711 22249 5904 480
860 28264 26593 716
846 21352 22581 333
773 18646 29993 592
239 27384 17980 277
112 21688 12194 628
1000 8267 4760 972
129 14540 11976 750
457 20839 4209 942
941 6316 17748 608
607 29138 27349 764
746 13379 22508 517
903 27984 18940 976
939 22366 4915 316
16 6290 24920 214
776 29144 21162 335
629 23585 24012 770
404 26569 1787 889
188 21010 5784 771
420 11933 1630 793
20 3322 4490 653
403 4958 17405 338
57 14418 8151 722
750 6184 17517 129
740 13378 2704 500
642 11495 17453 420
901 17181 10523 157
447 18321 1128 443
21 19466 3066 232
54 22424 24133 733
154 23462 8175 474
409 21903 29470 858
136 29148 10253 817
639 17130 15202 131
91 13539 15923 997
506 26537 370 275
589 5527 4685 905
854 21642 23526 781
304 21008 12178 194
569 11712 4163 201
824 4057 4441 694
702 21462 440 953
269 9144 27262 395
471 13127 21439 815
218 21059 9642 586
799 28072 15140 606
608 20102 28065 956
586 11176 330 392
229 20761 12931 225
264 23282 25625 869
193 19884 8553 627
195 10664 23030 810
671 11765 10021 514
929 20470 10559 174
309 2888 6424 618
464 18246 5617 588
768 13693 7988 699
211 3031 15044 433
912 12403 22396 903
993 14459 14126 328
561 16664 4218 288
27 16842 21898 210
847 26687 12551 207
495 22310 19338 996
575 23262 9796 391
782 20785 15815 580
888 7046 2042 561
744 6233 25187 905
135 25656 10857 126
138 513 4868 180
425 21336 5288 630
773 6271 4144 283
30 5699 20600 951
788 22164 19984 527
978 10027 2701 474
141 25073 7249 464
237 21275 5728 906
858 11375 24579 977
820 2282 3613 237
58 8944 27637 907
315 29439 26926 188
850 22454 21154 477
577 21179 4791 614
596 21717 11167 891
907 8833 12234 831
818 116 18469 781
327 2989 28774 620
441 29736 17276 353
755 20107 3405 626
641 26604 17895 228
833 7570 8567 243
715 9781 5053 885
461 22973 13474 481
494 1630 5156 923
841 1820 1611 737
179 16281 26791 802
208 28933 29808 281
393 11200 1500 250
407 22225 17122 571
387 6474 13784 207
195 8199 21943 121
74 4689 9868 786
403 8914 10545 550
847 28582 16016 115
935 2054 1112 585
424 15327 3050 702
780 8183 4891 674
932 28518 301 760
986 3127 18832 701
46 20059 8018 196
440 22800 27268 974
155 15638 13662 671
1000 7089 11244 147
21 26424 16205 848
281 1152 6005 470
713 19564 2084 813
984 29051 10499 836
882 8381 25859 356
7 21673 22069 585
159 5162 29869 513
1000 25158 13671 853
38 13372 1417 317
502 17670 22215 290
631 20567 14043 243
130 21614 11900 923
918 17011 4286 581
789 18924 16577 425
346 6947 6265 794
55 8698 23943 187
279 6444 16296 189
486 26132 17056 992
619 19468 10517 245
151 27101 10448 518
655 28068 22298 178
491 20443 19743 146
466 15189 29225 428
586 6520 9298 598
897 10216 10001 454
123 2383 25413 327
767 28742 24600 397
412 17917 25955 383
802 17066 23040 204
457 13292 1639 778
799 15922 8779 302
671 24736 23951 565
533 13567 27323 755
47 22656 6418 899
785 11813 16417 151
597 15548 13125 342
446 14398 2245 368
779 29902 13653 426
238 22929 10066 521
75 22260 16858 668
913 10607 11616 515
629 27312 1596 627
655 18413 2249 262
676 6252 2357 957
102 23121 16349 125
915 28880 25817 542
517 17504 11671 650
438 27943 14716 572
66 12351 26504 740
877 8694 6931 263
397 15531 15793 989
46 2073 17164 370
528 15148 14173 687
820 18794 2066 419
362 11337 4416 639
703 12799 13297 663
695 10768 14261 280
355 20571 2572 814
40 24751 10170 871
425 16086 10682 839
638 27304 6034 868
853 4458 5836 515
861 3566 9944 398
956 915 10472 676
83 27002 14001 317
118 13437 16683 747
314 2509 10885 176
81 11738 28316 962
845 21718 22855 830
940 6049 11306 991
448 16800 7716 120
251 4403 10430 383
946 27333 22270 800
54 15415 14803 227
373 1368 9125 560
103 5453 6416 421
340 11299 18404 698
368 19280 21986 997
347 26815 20418 765
596 27598 22644 297
650 20893 11383 424
414 25056 28091 203
