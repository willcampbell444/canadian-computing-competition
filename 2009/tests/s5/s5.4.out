345870
13480
